tardis/opacities/opacity_state.py

```diff
@@ -4,6 +4,79 @@
 
 from tardis.opacities.tau_sobolev import calculate_sobolev_line_opacity
 from tardis.transport.montecarlo.configuration import montecarlo_globals
+from tardis.opacities.continuum.continuum_state import ContinuumState
+from tardis.opacities.macro_atom.macroatom_state import MacroAtomState
+
+
+class OpacityStatePython:
```

Comment:
rename this to `OpacityStatePython` to `OpacityState` and rename the other one to numba